css/styles.css

```diff
@@ -183,6 +187,35 @@ nav li {
 	background-image: url("../IMG/comingSoon.jpg");
 }
 
+.contactMe {
+	overflow: auto;
+	background-color: #484349;
+	padding: 5% 5% 2% 5%;
+	color: #F5313E;
+}
+
+.contactMe div {
+	padding: 5% 0 5% 0;
+	text-align: center;
+}
+
+.contactIcon {
+	width: 60px;
```

Comment:
This applies to all the icons - they all work as hyperlinks in their entirety, however the hyperlinks extend beyond the dimensions of the icon. It might be worth trying a width: 60px on the <a> element to correct?

Reply:
Doesn't affect usability, so to be addressed later